proto/validator/accounts/v2/keymanager.proto

```diff
@@ -44,6 +45,18 @@ message SignRequest {
     // signing domain as well as the signing root of the data structure
     // the bytes represent.
     bytes signing_root = 2;
+
+    // Signature domain and the beacon chain objects to allow server to verify
+    // the contents and to prevent slashing.
+    bytes signature_domain = 3;
+    oneof object {
+        ethereum.eth.v1alpha1.BeaconBlock block = 4;
```

Comment:
I can imagine we'll be changing these often, maybe adding more as needed. Perhaps we should use wire_type ranges for these

Reply:
Updated. Let me know if that's not right @rauljordan